docs/install.md

```diff
@@ -8,14 +8,16 @@
 ## AppDaemon install
 
 - Install AppDaemon add-on [https://github.com/hassio-addons/addon-appdaemon](https://github.com/hassio-addons/addon-appdaemon)
-    - You will find the appdaemon.yaml file in addon_configs/a0d7b954_appadaemon
-        - Add to the appdaemon: setion **apps_dir** which should point to /homeassistant/appdaemon/apps
+    - You will find the appdaemon.yaml file in addon_configs/a0d7b954_appdaemon
```

Comment:
@springfall2008 can you confirm please that the extra 'a' in `appadaemon` is a mistake, and that it should be `appdaemon` here as I've fixed it? I'm not on the latest version so can't check for myself, but I'm assuming it's currently spelt wrong here.

Reply:
Yes it's a typo